Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -653,4 +636,15 @@ void SoftwareGLContext::present()
     m_rasterizer.blit_to(*m_frontbuffer);
 }
 
+
+bool SoftwareGLContext::currently_in_draw_state()
```

Comment:
This definitely needs a better name. The current one makes it sound like it just answers the question "are you currently in draw state?" but it will actually set an error.

Perhaps something a bit more imperative like.. `validate_draw_state()`?

Reply:
Agreed. How about `validate_is_in_draw_state`. That way we both have `validate` and we have the boolean `is_in` there too. If not, `validate_draw_state` works fine for me.

Reply:
`validate_is_in_draw_state` doesn't sound right. If it's between that and `validate_draw_state` I prefer the latter.

How about.. `validate_currently_in_draw_state()`?

Reply:
> How about.. validate_currently_in_draw_state()?

Works for me!